docs/downloading-media/downloading-multiple-files.md

```diff
@@ -48,8 +47,14 @@ class DownloadMediaController
         // Download the files associated with the media in a streamed way.
         // No prob if your files are very large.
         return MediaStream::create('my-files.zip')
-            ->useZipOptions(function(ArchiveOptions $zipOptions) {
-                $zipOptions->setZeroHeader(true);
+            ->useZipOptions(function(&$zipOptions) {
+                // ZipStream ^3.0 uses array
+                /** @var array $zipOptions */
+                $zipOptions['defaultEnableZeroHeader'] = true;
+
+                // ZipStream ^2.0 uses \ZipStream\Option\Archive
```

Comment:
Could we make this code work for both versions? Maybe using the `class_exists` function?

Reply:
You are rigth, it was just an example, I don't think someone will do it for both versions, I will change it